extensions/amp-script/validator-amp-script.protoascii

```diff
@@ -24,41 +24,47 @@ tags: {
   }
   attr_lists: "common-extension-attrs"
 }
-# Temporarily disable until interaction of amp-script and signed exchanges has
-# been security reviewed.
-#tags: {  # <amp-script> (local script)
-#  html_format: AMP
-#  tag_name: "SCRIPT"
-#  spec_name: "amp-script extension local script"
-#  requires_extension: "amp-script"
-#  attrs: {
-#    name: "target"
-#    mandatory: true
-#    value: "amp-script"
-#    dispatch_key: NAME_VALUE_DISPATCH
-#  }
-#  attrs: {
-#    name: "type"
-#    mandatory: true
-#    value_casei: "text/plain"
-#  }
-#  attr_lists: "mandatory-id-attr"
-#  attr_lists: "nonce-attr"
-#  cdata: {
-#    max_bytes: 10000
-#    max_bytes_spec_url: "https://amp.dev/documentation/components/amp-script#faq"
-#    blacklisted_cdata_regex: {
-#      regex: "<!--"
-#      error_message: "html comments"
-#    }
-#  }
-#  spec_url: "https://amp.dev/documentation/components/amp-script"
-#}
+tags: {  # <amp-script> (local script)
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-script extension local script"
+  requires_extension: "amp-script"
+  attrs: {
+    name: "target"
+    mandatory: true
+    value: "amp-script"
+    dispatch_key: NAME_VALUE_DISPATCH
+  }
+  attrs: {
+    name: "type"
+    mandatory: true
+    value_casei: "text/plain"
+  }
+  attr_lists: "mandatory-id-attr"
+  attr_lists: "nonce-attr"
+  cdata: {
+    max_bytes: 10000
+    max_bytes_spec_url: "https://amp.dev/documentation/components/amp-script#faq"
+    blacklisted_cdata_regex: {
+      regex: "<!--"
+      error_message: "html comments"
+    }
+  }
+  spec_url: "https://amp.dev/documentation/components/amp-script"
+}
 tags: {  # <amp-script>
   html_format: AMP
   tag_name: "AMP-SCRIPT"
   disallowed_ancestor: "AMP-SCRIPT"
   requires_extension: "amp-script"
+  attrs: {
+    name: "max-age"
+    value_regex: "[0-9]+"
```

Comment:
Maybe `0|[1-9][0-9]*` to disallow 0-prefix numbers? :man_shrugging:

Reply:
This is consistent with a lot of other `value_regex` number values though I don't feel strongly either way.